src/proxies/usersapi.js

```diff
@@ -0,0 +1,26 @@
+const platformClient = require('purecloud-platform-client-v2');
+
+const createUser = async (userInfo) => {
+  apiInstance = new platformClient.UsersApi();
+
+  const user = {
+    name: userInfo.NAME,
+    email: userInfo.EMAIL,
+    password: userInfo.password,
+  };
+
+  try {
+    const result = await apiInstance.postUsers(user);
```

Comment:
```suggestion
    return await apiInstance.postUsers(user);
```

Reply:
Yeah.  I was a little wordy at times on the returns because I had the different console.log statements to help debug.  I will clean this up.